### PR TITLE
APEXMALHAR-1859 tracking violation per malhar module

### PR DESCRIPTION
**Pull request by otterc**

### CODE REVIEW

**Comment by tweise** [comment]
Should we include the kafka module?

File: pom.xml [hunk 1]
```diff
@@ -90,13 +90,6 @@
           </excludes>
         </configuration>
       </plugin>
-      <plugin>
```
Comment (tweise) [inline pom.xml:93]: Does this mean checkstyle is turned off by default? Wouldn't it be better to have it default to no violations with console output?

**Comment by otterc** [comment]
Addressed review comments

**Comment by otterc** [comment]
Squashed the changes

**Pull request merged**